lib/entity.js

```diff
@@ -121,20 +187,12 @@ Entity.prototype.trimSnapshot = function trimSnapshot (snapshot) {
   return snapshot;
 };
 
-Entity.digestMethod = function (type, fn) {
-  if ( ! type) throw new EntityError('type is required for digest method definitions');
-  if ( ! fn) throw new EntityError('a function is required for digest method definitions');
-  if ( ! fn.name) throw new EntityError('Anonmyous functions are not allowed in digest method definitions. Please provide a function name');
-  type.prototype[fn.name] = function () {
-    var digestArgs = Array.prototype.slice.call(arguments);
-    digestArgs.unshift(fn.name);
-    Entity.prototype.digest.apply(this, digestArgs);
-
-    var methodArgs = Array.prototype.slice.call(arguments);
-    return fn.apply(this, methodArgs);
-  };
-};
-
+/**
+ * Remove the internal sourced properties from the passed snapshot.
+ *
+ * [CONFIRM] based on https://github.com/mateodelnorte/sourced/search?utf8=%E2%9C%93&q=digestMethod
```

Comment:
I searched for any mention of this method using https://github.com/mateodelnorte/sourced/search?utf8=%E2%9C%93&q=digestMethod and did not see any results. Is this correct? Can this method be removed? Note: this would invalidate [my previous PR](https://github.com/mateodelnorte/sourced/pull/12)